_includes/sidebar.html

```diff
@@ -20,6 +20,11 @@
         Questions
     </a>
 
+    <a href="https://github.com/php-fig/fig-standards/blob/master/index.md" class="button">
```

Comment:
Oh is that where it lives! With no link to it I had no idea how else to get to that page. :-)  (That URL is what's linked from the Wiki right now.)